src/js/RoversList/RoversList.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+
+import { styled } from '@material-ui/core/styles';
+import MuiExpansionPanel from '@material-ui/core/ExpansionPanel';
+import MuiExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import MuiExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormHelperText from '@material-ui/core/FormHelperText';
+
+import Slider from '@material-ui/core/Slider';
+import Typography from '@material-ui/core/Typography';
+import Tooltip from '@material-ui/core/Tooltip';
+
+import { object, func } from 'prop-types';
+
+const NasaExpansion = styled(MuiExpansionPanel)({
+  boxShadow: 'none',
+});
+
+const NasaCameras = styled(MuiExpansionPanelDetails)({
+  display: 'flex',
+  flexDirection: 'column',
+  border: '1px solid grey',
+});
+
+class RoversList extends Component {
+  render() {
+    const { elem, handleChange, handleSolChange, optionFlags, solsRange } = this.props;
+
+    return (
+      <div key={elem.rover} className="rover-info">
+        <NasaExpansion square expanded={optionFlags[elem.rover]}>
+          <MuiExpansionPanelSummary aria-controls="panel1d-content" id="panel1d-header">
+            <FormControlLabel
+              control={
+                <Checkbox
+                  value={elem.rover}
+                  checked={optionFlags[elem.rover]}
+                  onChange={() => handleChange(elem.rover)}
```

Comment:
It will be better to create this arrow func outside cause otherwise it will create new function each render

Reply:
I don't really think so. You are right, it will be created any time component rerenders. But it's much more readable than creating functions returning functions.